src/openms/source/FEATUREFINDER/FeatureFinderAlgorithmPicked.cpp

```diff
@@ -242,10 +242,10 @@ namespace OpenMS
     //new scope to make local variables disappear
     {
       startProgress(0, intensity_bins_ * intensity_bins_, "Precalculating intensity scores");
-      double rt_start = map_.getMinRT();
-      double mz_start = map_.getMinMZ();
-      intensity_rt_step_ = (map_.getMaxRT() - rt_start) / (double)intensity_bins_;
-      intensity_mz_step_ = (map_.getMaxMZ() - mz_start) / (double)intensity_bins_;
+      double rt_start = map_.spectrumRanges().byMSLevel(1).getMinRT();
```

Comment:
I think this is now much better and less error prone.
We clearly express that only MS1 ranges are considered.